test/groovy/CloudFoundryDeployTest.groovy

```diff
@@ -301,14 +306,16 @@ class CloudFoundryDeployTest extends BasePiperTest {
 
         assertThat(shellRule.shell, hasItem(containsString('cf login -u "test_cf" -p \'********\' -a https://api.cf.eu10.hana.ondemand.com -o "testOrg" -s "testSpace"')))
         assertThat(shellRule.shell, hasItem(containsString("cf blue-green-deploy testAppName --delete-old-apps -f 'test.yml'")))
-        assertThat(shellRule.shell, not(hasItem(containsString("cf stop testAppName-old"))))
+        assertThat(shellRule.shell, not(hasItem(containsString("cf stop testAppName-old &> 1-cfStopOutput.txt"))))
```

Comment:
I think 1-cfStopOutput.txt will never be in, as by default a random value will be generated.

Reply:
done

Reply:
I check for "cf stop testAppName-old". That should be ok?